core-blocks/image/edit.js

```diff
@@ -346,6 +346,9 @@ class ImageEdit extends Component {
 							const minWidth = imageWidth < imageHeight ? MIN_SIZE : MIN_SIZE * ratio;
 							const minHeight = imageHeight < imageWidth ? MIN_SIZE : MIN_SIZE / ratio;
 
+							const showRightHandle = ( isRTL && ( align === 'left' || align === 'center' ) || ! isRTL && align !== 'right' );
```

Comment:
The logic here feels a bit hard to read. Any way to make it more readable? Guessing it could read something like, if it's aligned centre both handles should show, otherwise show right or left depending on `isRTL`.

Reply:
There also seems to be a listing error:

```
[0] /home/travis/build/WordPress/gutenberg/core-blocks/image/edit.js
[0]   349:40  error  Unexpected mix of '&&' and '||'  no-mixed-operators
[0]   349:86  error  Unexpected mix of '&&' and '||'  no-mixed-operators
[0]   349:86  error  Unexpected mix of '||' and '&&'  no-mixed-operators
[0]   349:97  error  Unexpected mix of '||' and '&&'  no-mixed-operators
[0]   350:39  error  Unexpected mix of '&&' and '||'  no-mixed-operators
[0]   350:59  error  Unexpected mix of '&&' and '||'  no-mixed-operators
[0]   350:59  error  Unexpected mix of '||' and '&&'  no-mixed-operators
[0]   350:70  error  Unexpected mix of '||' and '&&'  no-mixed-operators
[0] 
[0] ✖ 8 problems (8 errors, 0 warnings)
```

Reply:
> Unexpected mix of '&&' and '||'  no-mixed-operators

Ha, why am I not seeing these here... Also... "unexpected"... in parentheses..? What is that rule, lol.

Yeah, it is a bit hard to read, can change that to a "proper" `if()` block. I know, wordy, but oh well...

Reply:
Actually... How do you "unblock" that linter when:

```if ( true && ( true || false ) )```

This looks like a bug there?

Reply:
Looks like it's catching the last bit: `|| ! isRTL && align !== 'right'`.

Reply:
In any case it wouldn't hurt to be a bit more readable :)

Reply:
Yep, an `if()` block would be better there.